pages/[network]/pull-request.tsx

```diff
@@ -221,7 +221,7 @@ export default function PullRequestPage() {
                       <Button
                         className="read-only-button text-nowrap"
                         onClick={handleShowModal}
-                        disabled={isCreatingReview || isCancelling || isMakingReady}
+                        disabled={isCreatingReview || isCancelling || isMakingReady || !user?.login }
```

Comment:
... damn. User does not need a connected github to make a review. will need to change this on the next sprint